README.md

```diff
@@ -2,61 +2,126 @@
 
 The Substreams:SQL sink helps you quickly and easily sync Substreams modules to a PostgreSQL or Clickhouse database.
 
-### Quickstart
+## Substreams Output Formats
+
+The `substreams-sink-sql` supports two different Substreams output formats, each with distinct advantages:
+
+### Relational Mappings (Recommended)
+
+**How it works:** Tables and rows are extracted dynamically from Protobuf messages using annotations for table and relation mappings. This approach leverages Substreams' built-in relational mapping capabilities.
+
+**Pros:**
+- **Less development work** - No need to manually emit database changes in your Substreams code
+- **Automatic schema inference** - Tables and relationships are derived from your Protobuf definitions
+- **Type safety** - Protobuf annotations ensure data consistency
+- **Easier maintenance** - Schema changes are managed through Protobuf definitions
+
+**Cons:**
+- **Less control** - Limited flexibility in how data is structured in the database
+- **Learning curve** - Requires understanding of Protobuf annotations and relational mappings
```

Comment:
Remove that one, replace by `**Insert Only** - Does not support update/delete of rows, it's an insert-only method of ingestion`